bot/cogs/queue.py

```diff
@@ -200,3 +212,100 @@ async def cap_error(self, ctx, error):
             missing_perm = error.missing_perms[0].replace('_', ' ')
             embed = self.bot.embed_template(title=f'Cannot change queue capacity without {missing_perm} permission!')
             await ctx.send(embed=embed)
+
+    async def is_banned(self, guild, user):
```

Comment:
@b3none I was thinking this might be a better in the DB helper class? To abstract away the management of unban times, since it makes the check if someone's duration has ended in this function. What do you think.

Reply:
Moved